packages/ui/src/forum/components/threads/ThreadItem.tsx

```diff
@@ -109,18 +97,6 @@ const ThreadItemText = styled(TextMedium)`
   overflow: hidden;
```

Comment:
A lot more work is needed on the thread browser and it's items, but it should probably be done on follow up PRs. That said the items should at least make cap the post text to 4 lines. One way to do this would be to add:
```ts
DotsNLines: (n: number) => css`
  display: -webkit-box;
  -webkit-line-clamp: ${n};
  -webkit-box-orient: vertical;
  -moz-box-orient: vertical;
  overflow: hidden;
`,
```
to `Overflow` into `@/common/constants/styles`.

So this would become:
```suggestion
  ${Overflow.DotsNLines(4)};
```